scrc/preprocessors/extractors/spider_specific/section_splitting_functions.py

```diff
@@ -25,13 +27,104 @@ def XX_SPIDER(decision: Union[bs4.BeautifulSoup, str], namespace: dict) -> Optio
     pass
 
 
+def VD_Omni(decision: Union[bs4.BeautifulSoup, str], namespace: dict) -> Optional[Dict[Section, List[str]]]:
+
+
+    def get_paragraphs(soup):
+        """
+        Get composition of the decision
+        :param soup:
+        :return:
+        """
```

Comment:
This should get the paragraphs (number of sentences delimited by line breaks normally or they might be in different html tags).